src/main/java/no/sikt/nva/UnZipper.java

```diff
@@ -34,7 +34,6 @@ public static List<File> extractResourceDirectories(String pathToZip, String des
             var unzippedFile = unzip(fileToUnzip, destinationFile);
             return Arrays.stream(Objects.requireNonNull(unzippedFile.listFiles())).collect(Collectors.toList());
         } catch (Exception e) {
-            logger.error(e.getMessage());
```

Comment:
this one is not needed.

Reply:
Det var den som avslørte at unzippinga trynet til meg?